test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -303,15 +303,15 @@ std::string RandomCode::generate(int _maxOpNumber, RandomCodeOptions _options)
 	return code;
 }
 
-std::string RandomCode::randomUniIntHex(u256 _maxVal)
+std::string RandomCode::randomUniIntHex(u256 _maxVal, u256 const _minVal)
 {
 	if (_maxVal == 0)
 		_maxVal = std::numeric_limits<uint64_t>::max();
 	refreshSeed();
 	int rand = randUniIntGen() % 100;
 	if (rand < 50)
-		return toCompactHexPrefixed((u256)randUniIntGen() % _maxVal);
-	return toCompactHexPrefixed((u256)randUInt64Gen() % _maxVal);
+		return toCompactHexPrefixed(_minVal + (u256)randUniIntGen() % _maxVal, 1);
```

Comment:
This is not correct at all, you'll get numbers between `_minVal` and `_minVal + maxVal - 1`

The optimal way I suggest is to create `boostIntDistrib(_minVal, _maxVal)` here and use it